espnet2/curriculum/curriculum_sampler.py

```diff
@@ -80,8 +81,7 @@ def __init__(
         for id in tasks:
             task_keys[tasks[id][0]].append(id)
             
-        # Sort samples in descending order
-        sorted_task_keys = [sorted(t, key=lambda k: first_utt2shape[k][0]) for t in task_keys]
+        sorted_task_keys = task_keys
```

Comment:
removed sorting since we shuffle them later anyways

Reply:
The training examples are being shuffled within the batch. We still need overall sorting, I think.